h2/connection.py

```diff
@@ -859,24 +859,40 @@ def send_data(self, stream_id, data, end_stream=False):
         :param end_stream: (optional) Whether this is the last data to be sent
             on the stream. Defaults to ``False``.
         :type end_stream: ``bool``
+        :param pad_length: (optional) Length of the padding to apply to the
+            data frame. Defaults to ``None`` for no use of padding. Note that
+            a value of ``0`` results in padding of length ``0``
+            (with the "padding" flag set on the frame).
+        :type pad_length: ``int``
+            .. versionadded:: 2.6.0
```

Comment:
If you build this the docs locally, the directive doesn’t show up correctly. I confess I don’t fully understand how Sphinx directives work, but this block of RST seems to do the trick:

```rst
        :param pad_length: (optional) Length of the padding to apply to the
            data frame. Defaults to ``None`` for no use of padding. Note that
            a value of ``0`` results in padding of length ``0``
            (with the "padding" flag set on the frame).

            .. versionadded:: 2.6.0

        :type pad_length: ``int``
```